### ISSUE TITLE
Does it work with NextJS?

### ISSUE THREAD

**Issue by mt-empty** [velut/node-query-registry#239]
Hi,

I'm trying to get this library working with NextJS in the browser.
I'm getting an import error:
```javascript
error - ./pages/index.tsx:2:0
Module not found: Can't resolve 'query-registry'
  1 | import PageLoader from "../components/pageLoader"
> 2 | import { getPackageManifest } from "query-registry";
  3 | 
  4 | export default function Profile() {
  5 | 

https://nextjs.org/docs/messages/module-not-found
```

Please note that it is usable server side, but on the client side as stated in the readme.md.

Any idea why it doesn't work on the client side?

**Comment by velut** [comment]
Hi @mt-empty,

It should work with Next.js, but it seems like it doesn't starting with version `2.3.0`; I'll have to look into the problem.

In the meantime, you can downgrade to version [`2.2.0`](https://www.npmjs.com/package/query-registry/v/2.2.0) and it should work; see this example https://codesandbox.io/s/serene-sid-fom5en?file=/pages/index.js

Also, remember to use the `cloudflareRegistry` on the client since it supports CORS.

**Comment by velut** [comment]
The problem was that the build system changed the output name from `index.esm.js` to `query-registry.esm.js`, while `package.json` values stayed the same.

Can you try installing [prerelease version 2.5.0-0](url)https://www.npmjs.com/package/query-registry/v/2.5.0-0] and see if it works for you?

Thanks.

**Comment by velut** [comment]
I've released version `2.5.0` with the fixed ESM export declarations.
